packages/firebase_ml_vision/test/firebase_performance_e2e.dart

```diff
@@ -0,0 +1,46 @@
+import 'dart:io';
```

Comment:
I think you meant to name this firebase_ml_vision_e2e.dart

Reply:
Yea I noticed that was one of the problems that caused the failing test. I fixed it in https://github.com/FirebaseExtended/flutterfire/pull/1363